updates/13xx_petguardians_update.sql

```diff
@@ -0,0 +1,33 @@
+-- These are but a fraction of the guardian pets, but most are with engi and need to be scripted. This will do for now.
+-- I am fixing the core to handle guardians by their values, check https://github.com/cmangos/mangos-classic/pull/116 for reference.
+
+-- Fixed Health of Timbermaw Ancestor (15720)
+UPDATE creature_template SET MinLevelHealth=500, MaxLevelHealth=500 WHERE Entry=15720;
```

Comment:
We trust you that you did not invent those values but took them from various sources. :smile:
This brings the usual topic of sources reliability. One can find many sources for WoW content on the Internet or in books. Some are more reliable than other and it also depends on when said source was edited.

Usually, my trust priority is:
- Sniffs (correct patch > later patch)
- Official WoW Bestiary / Guides books
- Screenshots/Videos that have a valid timestamp
- WoW wiki / WoW pedia when cross checked with other sources
- wowhead and the like

And even with sniffs (that are Blizz actual values), we often misunderstand them. So *careful* is a keyword here.

Regarding Timbermaw Ancestor, I fixed the NPC some time ago when I first notice an issue with it while testing the pet patch from Cyberium. I used Bestiary values for it and the NPC stats are OK now. The 500 HP value in wow wiki are more than probably an estimation made by the player due to the fact that in 2006, players did not have access to the real HP values (they were not sent to the client). Thus, HP and other values were estimated by players or addons. The correct value for Timbermaw Ancestor is 427 HP. Usually, when seeing a nice flat value for HP, mana or damage, chances are good this is a estimation, not the actual value.